Remotion/ObjectBinding/Web/UI/Controls/Validation/ValidationFailureHandlingContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// using NUnit.Framework;
+//
+using System.Text;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.Utilities;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls.Validation
+{
+  public class ValidationFailureHandlingContext
+  {
+    public IBocList BocList { get; }
+
+    public IBocListValidationFailureRepository ValidationFailureRepository => BocList.ValidationFailureRepository;
+
+    private readonly StringBuilder _sb;
```

Comment:
```suggestion
    private readonly StringBuilder _errorMessage;
```
or `_errorMessageBuilder`

`sb` is only okay in very limited scenarios (mostly only places where we would use other abbreviations, too. `stringBuilder` is okay when it's in the very local scope and obvious what it is (e.g. the only purpose of the method is building that string). And even then, it's probably more of a legacy thing than good practice. For fields it should always be named, same as when we have a `string` or `int` field, which also have semantic names.